lib/palette/elastic_search/searchable.rb

```diff
@@ -139,12 +139,12 @@ def reindex!
                            tokenizer: 'keyword',
                            char_filter: %W(my_icu_normalizer company_name_trimmer)
                          },
-                         bigram: {
-                           tokenizer: 'bi_gram',
+                         ngram: {
```

Comment:
this change cloud not support current analyzer `bigram`, pls add `ngram` as different one.